pom.xml

```diff
@@ -58,9 +58,9 @@
             <version>2.1.1</version>
         </dependency>
         <dependency>
-            <groupId>com.github.lavalink-devs</groupId>
-            <artifactId>lavaplayer-youtube-source</artifactId>
-            <version>1.0.4</version>
+            <groupId>dev.lavalink.youtube</groupId>
+            <artifactId>v2</artifactId>
+            <version>b2c8c070cdc4caa1fa93d0d7ac3f9d468bfa27a8-SNAPSHOT</version>
```

Comment:
Latest tag is [1.3.0](https://github.com/lavalink-devs/youtube-source/tree/1.3.0), I can switch from the commit `SNAPSHOT` version to `1.3.0` if you prefer, didn't test it however as I didn't initially plan to open a PR with it.